.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -28,6 +29,7 @@ body:
       description: Which server version version you using? If your server version is not listed, it is not supported. Update to a supported version first.
       multiple: false
       options:
+        - '1.17.1'
```

Comment:
1.17.1 isn't released yet

Reply:
shhh, we're future proof. Also coming soonTM